File: backend/src/main/java/com/tanerkaynar/nexuserp/model/Kullanici.java
package com.tanerkaynar.nexuserp.model;

import jakarta.persistence.*;
import lombok.Data;
import java.time.LocalDateTime;

@Data
@Entity
@Table(name = "kullanicilar")
public class Kullanici {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "kullaniciid")
    private Integer kullaniciid;

    @Column(name = "kullaniciadi", nullable = false, unique = true)
    private String kullaniciadi;

    @Column(name = "rol", nullable = false)
    private String rol;

    @Column(name = "aktifmi", nullable = false)
    private Boolean aktifmi = true;

    @Column(name = "kayittarihi", insertable = false, updatable = false)
    private LocalDateTime kayittarihi;

    @Column(name = "parolahash", nullable = false)
    private String parolahash;

    @OneToOne
    @JoinColumn(name = "personelid", unique = true)
    private Personel personel;
}

File: client-desktop/NexusERP/Models/Kullanici.cs
using System;

namespace NexusERP.Models
{
    public class Kullanici
    {
        public int KullaniciID { get; set; }
        public string KullaniciAdi { get; set; }
        public string Parola { get; set; }
        public string Rol { get; set; }
        public bool AktifMi { get; set; }
        public DateTime KayitTarihi { get; set; }
        public string ParolaHash { get; set; }
        public Personel Personel { get; set; }
    }
}